src/NServiceBus.AwsLambda.SQS.AcceptanceTests/AwsLambdaSQSEndpointTestBase.cs

```diff
@@ -133,6 +136,12 @@ public static IAmazonSQS CreateSQSClient()
             return new AmazonSQSClient(credentials);
         }
 
+        public static IAmazonSimpleNotificationService CreateSNSClient()
```

Comment:
If we bump the minimum required dependency of the SDK types to versions that reference [3.7.106 of AWSSDK.Core](https://www.nuget.org/packages/AWSSDK.Core/3.7.106) or higher we can leave the settings as is. Otherwise we would have to introduce a client factory similar to

https://github.com/Particular/NServiceBus.AmazonSQS/blob/master/src/NServiceBus.Transport.SQS.Tests/ClientFactories.cs

Reply:
Is your suggestion based on having the ability to set the HttpClientCacheSize?

Reply:
The cache would be correct as of that version and we can remove the explicit setting to one. Same applies to the transport and dynamodb by the way.

Reply:
I bumped to the latest SQS transport, is that sufficient?  We don't currently have a direct dependency on AWSSDK.Core .